workers/unity/Packages/com.improbable.gdk.legacy/Editor/BuildSystem/Util/SpatialCommand.cs

```diff
@@ -19,9 +19,6 @@ public sealed class SpatialCommand
 
         internal const string usrLocalBin = "/usr/local/bin";
 
-        private string spatialLocation;
-        private string oldSpatialLocation;
-        private string discoveredLocation;
```

Comment:
I think there's an open ticket for this which you can close if its merged in here